### PR TITLE
Rt 131151 revive news

### PR DESCRIPTION
**Pull request by jkeenan**

### CODE REVIEW

**Comment by rspier** [comment]
Thanks Jim.  I'll send some comments later tonight.

File: docs/dev/perl5/news/2013/perl-5.16.3.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+<p>
+https://metacpan.org/release/RJBS/perl-5.16.3/
+</p>
+
+<p>
+SHA1 digests for this release are:
+</p>
+
+<p>
```
Comment (rspier) [inline docs/dev/perl5/news/2013/perl-5.16.3.html]: It's not necessary to wrap `<pre></pre>` in a `<p></p>`.

File: docs/dev/perl5/news/2013/perl-5.16.3.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+see the AUTHORS file in the Perl source distribution.
+</p>
+
+<p>
+The next major stable release of Perl 5, version 18.0, should appear in May
+2013.
+</p>
+
+<p>
+rjbs
```
Comment (rspier) [inline docs/dev/perl5/news/2013/perl-5.16.3.html]: It's a little weird how `rjbs` is just down here at the bottom.

**I** know it's his email signature, but it needs some sort of delimiter or explanation or styling.  See for example this one: https://dev.perl.org/perl5/news/2004/perl-5.8.3.html

File: docs/dev/perl5/news/2013/perl-5.16.3.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
+[% page.title = "Perl 5.16.3 Release Announcement" %]
+<meta charset="utf-8">
```
Comment (rspier) [inline docs/dev/perl5/news/2013/perl-5.16.3.html]: Is this actually necessary?

Our pages should already have 
    `<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />`

in them.

File: docs/dev/perl5/news/2013/perl-5.18.1.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+<p>
+https://metacpan.org/release/RJBS/perl-5.18.1/
+</p>
+
+<p>
+SHA1 digests for this release are:
+</p>
+
+<p>
```
Comment (rspier) [inline docs/dev/perl5/news/2013/perl-5.18.1.html]: In a previous file this was in a code font.

File: docs/dev/perl5/news/2013/perl-5.18.1.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+18 of Perl 5.
+</p>
+
+<p>
+You will soon be able to download Perl 5.18.1 from your favorite CPAN
+mirror or find it at:
+</p>
+
+<p>
+https://metacpan.org/release/RJBS/perl-5.18.1/
```
Comment (rspier) [inline docs/dev/perl5/news/2013/perl-5.18.1.html]: Should this be a proper link?

File: docs/dev/perl5/prep/inputs/5.16.2.source.txt [hunk 6]
```diff
@@ -0,0 +1,40 @@
+The Perl 5 development team is gratified to announce the release of
```
Comment (rspier) [inline docs/dev/perl5/prep/inputs/5.16.2.source.txt]: Can we find a different place to put the prep content?  Right now it's in the *serving* tree and it's kind of clutter.

Maybe a new git repository?  It doesn't have to be anywhere near this one.  I could even make a good argument for keeping it in the main perl5 repo.

I had thought we had built something into Combust for private directories, but I can't find it.  If you really want to keep them in this repo, then they should be in the same directory and something we're ok with serving.

Comment (rspier) [inline docs/dev/perl5/prep/inputs/5.16.2.source.txt]: These "working files" in the repo (and the .html) under prep are my biggest issues with it.  I can look the other way on the formatting comments above.

File: docs/dev/perl5/news/2013/perl-5.18.1.html (hunk 4 above)
Comment (jkeenan) [inline docs/dev/perl5/news/2013/perl-5.18.1.html]: I was trying to more or less match the style of the most recent page in this directory: http://dev.perl.org/perl5/news/2012/perl-5.16.2.html

**Comment by rspier** [comment]
Thanks!  Merged!

**Comment by jkeenan** [comment]
I spot-checked http://dev.perl.org/perl5/news/index.html and http://dev.perl.org/perl5/news/2017/perl-5.24.1.html.  Looks good.  Thanks for your assistance.

**Comment by abh** [comment]
Thank you, James!

**Pull request merged**